manifests/transport_https.pp

```diff
@@ -0,0 +1,8 @@
+# Allow APT to use repos served over HTTPS
+class apt::transport_https (
+  $ensure = 'present'
```

Comment:
This is missing validation. Right now someone can pass in `ensure => banana` and will cause Puppet to go poof.